chi2/chinotype.py

```diff
@@ -98,7 +98,11 @@ def __init__(self, listargs=[], args={}):
         self.crc_user = db['crc_user']
         self.crc_service =  db['crc_service_name']
         self.crc_pw = db['crc_pw']
+        self.branchnodes = db['chi_branchnodes']
```

Comment:
Config variable specifying code for choosing branch nodes from the i2b2 ontology